src/opnsense/mvc/app/controllers/OPNsense/Routing/forms/dialogEditGateway.xml

```diff
@@ -58,17 +58,17 @@
     </field>
     <field>
         <id>gateway_item.monitor_killstates</id>
-        <label>Kill states when down</label>
+        <label>Failover States</label>
```

Comment:
```suggestion
        <label>Failover States When Down</label>
```

could also be more explicit... looking for symmetry

Reply:
"Failover associated states"?

"when down" in the detailed description perhaps?

Reply:
I had "attached states" in the description before. Went through a few local revisions of wanting to make it clear in the label and not explaining again in the description. So associated or attached in the label, description already has "down" either way?

Reply:
Well, "failover" implies it, "down" could at least be part of the extended description. I think it doesn't hurt to explain again in any case